genai-perf/genai_perf/export_data/console_exporter.py

```diff
@@ -66,6 +66,15 @@ def export(self) -> None:
         # System metrics are printed after the table
         for metric in self._metrics.system_metrics:
             line = metric.name.replace("_", " ").capitalize()
+            if metric.name == "request_goodput":
+                if not self._args.goodput:
+                    continue
+                value = self._stats[metric.name]["avg"]
+                if value is None:
+                    value = "N/A"
```

Comment:
What if we make the value 0 in these cases? Then the code could be refactored to avoid this unique logic handling and continue. It would also make parsing a bit easier, because all of the parsed values would be ints or floats.

For example:
```
for metric in self._metrics.system_metrics:
           line = metric.name.replace("_", " ").capitalize()
           if metric.name == "request_goodput":
                if not self._args.goodput:
                    continue
           line += f" ({metric.unit}): {value}"
                    print(line)
                    continue
```

Then, you could set the value to 0 somewhere else if there's an error. You should log that error in the same place, if you're not already.

Reply:
Updated